### PR TITLE
Add libcurl error advice

### PR DESCRIPTION
**Pull request by jeffreypullin**
I could add more detailed info about taking anaconda off the path, but it seems like it is not the simplest thing in the world [to do](https://unix.stackexchange.com/questions/108873/removing-a-directory-from-path). (I just removed the part of my `.zshrc` which added it in the first place.)

Best!

### CODE REVIEW

File: building.Rmd [hunk 1]
```diff
@@ -242,3 +242,7 @@ R --version
     ```sh
     xcode-select --install
     ```
+
+*   libcurl >= 7.28.0 library and headers are required with support for https
+
+    Take anaconda off of your PATH
```
Comment (lionel-) [inline building.Rmd]:
```suggestion
    Take anaconda off of your PATH ([SO](https://stackoverflow.com/questions/38690232/installing-r-on-linux-configure-error-libcurl-7-28-0-library-and-headers-a)).
```

Comment (jeffreypullin) [reply building.Rmd]: That question has a lot of other answers aimed at Linux users (including the accepted answer). How about we link to the specific answer [here](https://stackoverflow.com/a/51318770) - though its not particularly detailed.

Comment (lionel-) [reply building.Rmd]: Good idea.

**Comment by lionel-** [comment]
Thanks for this inaugural contribution :)

**Pull request merged**